honeybadger-js.d.ts

```diff
@@ -40,7 +40,7 @@ declare module "honeybadger-js" {
         static onerror: boolean;
         static wrap<T extends Function>(func: T): T;
         static setContext<T extends Object>(context: T): Honeybadger;
-        static resetContext(): Honeybadger;
+        static resetContext(context?: T): Honeybadger;
```

Comment:
You forget to declare the generic like this `static resetContext<T extends Object>(context?: T): Honeybadger;`